config/default/manager_auth_proxy_patch.yaml

```diff
@@ -23,3 +23,4 @@ spec:
         args:
         - "--metrics-addr=127.0.0.1:8080"
         - "--enable-leader-election"
+        - "--runner-image=summerwind/actions-runner:v2.168.0"
```

Comment:
If `defaultRunnerImage` is updated, you don't need to specify this option. Could you delete this line please?

Reply:
Yeah, I was struggling a little bit with how to do the setting, maybe we should have it in the documentation somewhere.

With that being said, I think I can remove this line.

Reply:
I agree with you. I will add usage section to the document, thanks!